transport.go

```diff
@@ -13,8 +14,8 @@ type WrapErrorFunc func(error) interface{}
 type Transporter interface {
 	getDispatcher() (dispatcher, error)
 	getReceiver() (receiver, error)
-	Run() error
-	RunAsync() <-chan error
+	ReceiveFrames() error
+	ReceiveFramesAsync()
```

Comment:
Because we only call this function in one place, and we don't do anything with that anyway.

And external clients that were calling RunAsync() and using the errCh would be better off just spawning their own goroutine and doing whatever they want with the error (e.g. sending it on their own channel).